app/models/collection.js

```diff
@@ -30,11 +28,12 @@ class Collection {
             _.extend(this, props);
         }
     }
-    addBullet(bullet) {
-      this.bullets.push(bullet.id);
-      if (bullet.collections.indexOf(this.id) < 0) bullet.collections.push(this.id)
-      return Promise.all([this.save(), bullet.save()])
-      .catch(err => console.error('error ', err))
+    addBullet(bullet, index) {
+        var index = index || this.bullets.length;   //so we can preserver ordering in collections.bullet array
+        this.bullets = this.bullets.slice(0, index).concat(bullet.id).concat(this.bullets.slice(index));
```

Comment:
Do we need to be able to add bullets at specific positions?  Or are we always adding to the end?